src/GitHub.InlineReviews/Glyph/GlyphData.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using Microsoft.VisualStudio.Text;
+
+namespace GitHub.InlineReviews.Glyph.Implementation
+{
+    internal class GlyphData<TGlyphTag>
```

Comment:
Could we get some documentation comments on the `GitHub.InlineReviews.Glyph` classes? Just a brief description of what they represent.